portfolio/src/main/webapp/index.html

```diff
@@ -61,5 +61,34 @@ <h2>Hello, my name is Christian Liguori.</h2>
         </div>
       </div>
     </div>
+
+    <!-- Contact Form -->
+    <div id="contact-form-container">
+      <div class="container">
+        <div class="row">
+          <div class="col s10 offset-s1">
+            <h3>Get in touch!</h3>
+            <form 
```

Comment:
This is a really well-structured HTML form: the labels and for attributes are helpful!